.goreleaser.yml

```diff
@@ -0,0 +1,34 @@
+project_name: cacophony-config
+
+release:
+  github:
+    owner: TheCacophonyProject
+    name: cacophony-config
+  name_template: '{{.Tag}}'
+
+builds:
+- goos:
+  - linux
+  goarch:
+  - arm
+  goarm:
+  - "7"
+  main: ./cmd/config
+  ldflags: -s -w -X main.version={{.Version}}
+  binary: cacophony-config
```

Comment:
how about you call this `cacophony-config-import`. I think soon we'll also want a tool to get and set config items from the command line and `cacophony-config` would be a good name for that.